chain/x/zoracle/handler.go

```diff
@@ -148,6 +153,39 @@ func NewHandler(keeper Keeper) sdk.Handler {
 // 	return sdk.Result{Events: ctx.EventManager().Events()}
 // }
 
+func handleMsgCreateDataSource(ctx sdk.Context, keeper Keeper, msg MsgCreateDataSource) sdk.Result {
+	newDataSourceID := keeper.GetNextDataSourceID(ctx)
+
+	newDataSource := types.NewDataSource(
+		msg.Owner,
```

Comment:
Check that fee contains only valid denom

Reply:
another module doesn't check it